.swift-version

```diff
@@ -0,0 +1 @@
+5.3
```

Comment:
If we have the `#if swift(>=5.6)` in the package.swift do we still need this? Also where is this file documented?

Reply:
This controls the Swift version that SwiftFormat uses -- otherwise it will make changes that are invalid in 5.3. I'll see if I can put a comment or something in this file.

Reply:
Yep this file supports comments as long as the version number is on the first line

Reply:
Unfortunately adding the comment caused `pod spec lint` to fail so I had to remove it. On the bright side that means this file is the source-of-truth for Lottie's support Swift version across multiple different tools.